pandas/core/indexing.py

```diff
@@ -1920,11 +1920,34 @@ def _convert_key(self, key, is_setter=False):
 
 
 class _iAtIndexer(_ScalarAccessIndexer):
-    """Fast integer location scalar accessor.
+    """
+    Access a single value for a row/column pair by integer position.
 
-    Similarly to ``iloc``, ``iat`` provides **integer** based lookups.
-    You can also set using these indexers.
+    Similar to ``iloc``, in that both provide integer-based lookups. Use
```

Comment:
this also works on a Series, can you re-word to accomodate

Reply:
Great point, I specify that `iat` works for both a `DataFrame` and a `Series`. I also added an example for using it on `Series`.